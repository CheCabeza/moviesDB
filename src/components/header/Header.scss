@import '../../styles/colors';

*, *::after, *::before {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

.header {
    display: flex;
    flex-direction: row;
    align-items: center;
    width: 100%;
    height: auto;
    background: linear-gradient( #F4C95D, white);

    &__logo {
        margin: 10px;
        margin-left: 30px;
        width: 70px;
        height: 70px;
    }
}