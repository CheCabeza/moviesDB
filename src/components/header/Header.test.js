import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

import Header from './Header';

describe('Given a Details component', () => {
  test('Should render the Header component', () => {
    const header = render(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,

    );
    expect(header).toMatchSnapshot();
  });

  //   test('ImFilm should be clicked', () => {
  //     const { getByTestId } = render(
  //       <BrowserRouter>
  //         <Header />
  //       </BrowserRouter>,

//     );
//     const ImFilmButton = getByTestId('ImFilm');
//     fireEvent.click(ImFilmButton);
//   });
});
