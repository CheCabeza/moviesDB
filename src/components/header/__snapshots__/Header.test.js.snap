// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Details component Should render the Header component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <img
          alt="TMDB logo"
          class="header__logo"
          src="https://www.themoviedb.org/assets/2/v4/logos/v2/blue_square_2-d537fb228cf3ded904ef09b136fe3fec72548ebc1fea3fbbd1ad9e36364db38b.svg"
        />
      </header>
      <div>
        <a
          class="movieIcon"
          href="/"
        >
          <svg
            fill="currentColor"
            height="3em"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="3em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 2v12h16v-12h-16zM3 13h-2v-2h2v2zM3 9h-2v-2h2v2zM3 5h-2v-2h2v2zM12 13h-8v-10h8v10zM15 13h-2v-2h2v2zM15 9h-2v-2h2v2zM15 5h-2v-2h2v2zM6 5v6l4-3z"
            />
          </svg>
        </a>
        <a
          class="tvIcon"
          href="/tvshows"
        >
          <svg
            fill="currentColor"
            height="3em"
            stroke="currentColor"
            stroke-width="0"
            version="1.1"
            viewBox="0 0 16 16"
            width="3em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M15.331 4.502c-1.388-0.199-2.865-0.344-4.407-0.425l2.576-2.576-1-1-3.509 3.509c-0.328-0.006-0.659-0.009-0.991-0.009v0l-4-4-1 1 3.034 3.034c-1.889 0.066-3.693 0.227-5.365 0.467-0.43 1.683-0.669 3.543-0.669 5.498s0.239 3.815 0.669 5.498c2.244 0.323 4.724 0.502 7.331 0.502s5.087-0.179 7.331-0.502c0.43-1.683 0.669-3.543 0.669-5.498s-0.239-3.815-0.669-5.498zM13.498 13.666c-1.683 0.215-3.543 0.334-5.498 0.334s-3.815-0.119-5.498-0.334c-0.323-1.122-0.502-2.362-0.502-3.666s0.179-2.543 0.502-3.666c1.683-0.215 3.543-0.334 5.498-0.334s3.815 0.119 5.498 0.334c0.323 1.122 0.502 2.362 0.502 3.666s-0.179 2.543-0.502 3.666z"
            />
          </svg>
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <img
        alt="TMDB logo"
        class="header__logo"
        src="https://www.themoviedb.org/assets/2/v4/logos/v2/blue_square_2-d537fb228cf3ded904ef09b136fe3fec72548ebc1fea3fbbd1ad9e36364db38b.svg"
      />
    </header>
    <div>
      <a
        class="movieIcon"
        href="/"
      >
        <svg
          fill="currentColor"
          height="3em"
          stroke="currentColor"
          stroke-width="0"
          version="1.1"
          viewBox="0 0 16 16"
          width="3em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 2v12h16v-12h-16zM3 13h-2v-2h2v2zM3 9h-2v-2h2v2zM3 5h-2v-2h2v2zM12 13h-8v-10h8v10zM15 13h-2v-2h2v2zM15 9h-2v-2h2v2zM15 5h-2v-2h2v2zM6 5v6l4-3z"
          />
        </svg>
      </a>
      <a
        class="tvIcon"
        href="/tvshows"
      >
        <svg
          fill="currentColor"
          height="3em"
          stroke="currentColor"
          stroke-width="0"
          version="1.1"
          viewBox="0 0 16 16"
          width="3em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M15.331 4.502c-1.388-0.199-2.865-0.344-4.407-0.425l2.576-2.576-1-1-3.509 3.509c-0.328-0.006-0.659-0.009-0.991-0.009v0l-4-4-1 1 3.034 3.034c-1.889 0.066-3.693 0.227-5.365 0.467-0.43 1.683-0.669 3.543-0.669 5.498s0.239 3.815 0.669 5.498c2.244 0.323 4.724 0.502 7.331 0.502s5.087-0.179 7.331-0.502c0.43-1.683 0.669-3.543 0.669-5.498s-0.239-3.815-0.669-5.498zM13.498 13.666c-1.683 0.215-3.543 0.334-5.498 0.334s-3.815-0.119-5.498-0.334c-0.323-1.122-0.502-2.362-0.502-3.666s0.179-2.543 0.502-3.666c1.683-0.215 3.543-0.334 5.498-0.334s3.815 0.119 5.498 0.334c0.323 1.122 0.502 2.362 0.502 3.666s-0.179 2.543-0.502 3.666z"
          />
        </svg>
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
